pkg/antctl/command_definition.go

```diff
@@ -47,6 +47,13 @@ const (
 // as:
 //   antctl <commandGroup> <command>
 type commandGroup uint
+type OutputType uint
+
+const (
+	defaultType OutputType = iota
+	single
+	multiple
```

Comment:
I think it's worth to add comments for this values, it took me a while to understand, maybe others will have question too.

Reply:
Comments added.